packages/editor-ui/src/n8n-theme-variables.scss

```diff
@@ -62,13 +62,6 @@ $tag-text-color: var(--color-text-dark);
 $tag-close-background-color: var(--color-text-light);
 $tag-close-background-hover-color: var(--color-text-dark);
 
-// nodes
```

Comment:
@elsmr I removed these saas vars as is seems like they are not used anymore, do you think it's fine?